sass/type/_normalize.scss

```diff
@@ -129,14 +129,6 @@
 
   $name: $name or inspect.is-alias-for($source, $key, 'origin') or $key;
 
-  @if (meta.type-of($name) == 'string') {
-    $name: if(
-      string.index($name, ''),
-      string.quote($name),
-      string.unquote($name)
-    );
-  }
-
```

Comment:
It may have been a mistake to remove this… but at the time it seemed redundant, and didn't have any tests.

Reply:
hmm. without these lines, the first two tests in `Fonts` fail due to the extra quotes:
![image](https://user-images.githubusercontent.com/41588129/141302318-e8991706-4078-4238-97b9-b9ff8ca03926.png)

Reply:
super weird that they didn't fail for me. but good catch - @dvdherron can you add these lines back in? (since this is used a couple places, it could also become a function if you wanted)

Reply:
yeah, strange. And I will look into turning this into a function . . .